src/index.tsx

```diff
@@ -907,6 +911,11 @@ class App extends React.Component<{}, AppState> {
     if (savedState) {
       this.setState(savedState);
     }
+
+    if (!(savedState && savedState.name)) {
+      const name = `excalidraw-${getDateTime()}`;
```

Comment:
Why the name is changing here?

Reply:
This is meant to be the default namehttps://github.com/excalidraw/excalidraw/pull/145#discussion_r363108764
I will do this .